packages/playground/remote/src/lib/setup-fetch-network-transport.ts

```diff
@@ -27,7 +27,13 @@ export async function setupFetchNetworkTransport(playground: UniversalPHP) {
 	});
 
 	await playground.onMessage(async (message: string) => {
-		const envelope: RequestMessage = JSON.parse(message);
+		let envelope: RequestMessage;
+		try {
+			// PHP-WASM sends messages as strings, so we can't expect valid JSON.
+			envelope = JSON.parse(message);
+		} catch (e) {
+			return '';
+		}
```

Comment:
@adamziel I ended up wrapping this in a try/catch. If we expect json, we shouldn't throw an error.
If there is a plain text message, it's not intended for this code.